tensorflow/lite/micro/micro_allocator_test.cc

```diff
@@ -795,4 +795,71 @@ TF_LITE_MICRO_TEST(TestOperatorInputsNotInSubgraphInputs) {
   TF_LITE_MICRO_EXPECT_EQ(0, eval_tensors[4].data.uint8 - start);
 }
 
+TF_LITE_MICRO_TEST(TestTypicalFirstOpAndSecondOpWithScratchTensors) {
+  constexpr int nbr_tensors = 6;
```

Comment:
nit: `number_tensors` - makes this a little easier to read when variables are not abbreviated.

Reply:
Done